config/main.py

```diff
@@ -601,65 +601,109 @@ def neighbor(ipaddr_or_hostname, verbose):
 #
 
 @cli.group()
-def interface():
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.pass_context
+def interface(ctx, interface_name):
     """Interface-related configuration tasks"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {}
+    ctx.obj['config_db'] = config_db
+    ctx.obj['interface_name'] = interface_name
     pass
 
 #
-# 'shutdown' subcommand
+# 'startup' subcommand
 #
 
 @interface.command()
-@click.argument('interface_name', metavar='<interface_name>', required=True)
-@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
-def shutdown(interface_name, verbose):
-    """Shut down interface"""
-    if get_interface_mode() == "alias":
-        interface_name = interface_alias_to_name(interface_name)
-        if interface_name is None:
-            raise click.Abort()
-
-    command = "ip link set {} down".format(interface_name)
-    run_command(command, display_cmd=verbose)
+@click.pass_context
+def startup(ctx):
+    """Start up interface"""
+    config_db = ctx.obj['config_db']
+    interface_name = ctx.obj['interface_name']
```

Comment:
You removed the recent `interface_alias_to_name()` alias-to-name translation addition here and below. Just wanted to check if this was overlooked because it's new. Do we want to maintain the ability to bring ports up/down by their vendor alias? @lguohan: do you have an opinion on this?

Reply:
Thanks for the feedback. I believe it is better to retain the ability of using vendor alias as an option. I will see how to update the pull request to fix this issue.

Reply:
@jleveque I have added the support for the `get_interface_mode() == "alias"`